tools/gen_internal_conf/src/main.rs

```diff
@@ -300,6 +300,52 @@ fn main() {
         );
         assert!(kernel_heap_max_size >= kernel_heap_init_size);
 
+        // Check validity of cache and disk size in mount options
```

Comment:
This check code can be moved to line 478.

Reply:
Done